main.go

```diff
@@ -94,6 +104,108 @@ func showErrorMessages(action string, errorMessages map[string]error) error {
 	return nil
 }
 
+type ConfProfile struct {
+	RemoteConfiguration bool `json:"active"`
+	Compliance          bool `json:"compliance"`
+	Insights            bool `json:"insights"`
+	Remediation         bool `json:"remediations"`
+}
+
+// MarshalJSON will convert the ConfProfile to a more meaningful structure
+func (p ConfProfile) MarshalJSON() ([]byte, error) {
+	type P struct {
+		RemoteConfiguration bool `json:"Remote Configuration"`
+		Compliance          bool `json:"Compliance"`
+		Insights            bool `json:"Insights"`
+		Remediation         bool `json:"Remediations"`
+	}
+	return json.Marshal(P(p))
+}
```

Comment:
I don't believe that this function is necessary. BTW: The structure `ConfProfile` is actually defined twice.

Reply:
That's true, I add this to print out nicely as a JSON (change the Active parameter to one that had more meaning). This is no longer used as the output isn't print as JSON but could be useful to print it with the `--status`. I will remove it because it doesn't belong to this PR.